src/draft/NSG/Agent/LegalEntity/NonListedCompany/Establishment/Write.py

```diff
@@ -0,0 +1,1633 @@
+        "is not available",
+        max_length=250,
+        example="Digital company Y",
+    )
+    founding_date: date = Field(
+        ...,
+        title="Founding date",
+        description="The date of memorandum of association",
+        example=date(2022, 1, 1),
+    )
+    industry_sector: IndustrySector = Field(
+        ...,
+        title="Industry sector",
+        description="The industry fields where the company will run its business. The codes are based on the "
+        "Statistical classification of economic activities in the European Community, abbreviated as NACE.",
+        example=IndustrySector.NACE_62_01,
+    )
+    share_capital: float = Field(
+        ...,
+        title="Share capital",
+        description="The total value of the issued shares of the company",
+        example=1000.0,
+    )
+    capital_currency: ISO_4217_CurrencyCode = Field(
+        ...,
+        title="Capital currency",
+        description="The currency used for the share capital in ISO 4217 format",
+        example=ISO_4217_CurrencyCode.EUR,
+    )
+    settlement_deposit: Optional[float] = Field(
+        None,
+        title="Settlement deposit",
+        description="The amount of the deposit paid during the establishment of a "
+        "company",
+        example=1000.0,
+    )
+    deposit_currency: Optional[ISO_4217_CurrencyCode] = Field(
+        None,
+        title="Deposit currency",
+        description="The currency used for the paying the settlement deposit in ISO "
+        "4217 format",
+        example=ISO_4217_CurrencyCode.EUR,
+    )
+    settlement_date: Optional[date] = Field(
+        None,
+        title="Settlement date",
+        description="The date of memorandum of association",
+        example=date(2022, 2, 1),
+    )
+    country_of_residence: Optional[ISO_3166_1_Alpha_3] = Field(
+        None,
+        title="Country of residence",
+        description="The company's current country of the residence in the three "
+        "character (Alpha-3) format if it already exists abroad",
+        example=ISO_3166_1_Alpha_3.USA,
+    )
+
+
+class ShareSeries(CamelCaseModel):
+    share_series_class: ShareSeriesClass = Field(
+        ...,
+        title="Share series class",
+        description="The type of the share series of a company",
+        example=ShareSeriesClass.A,
+    )
+    number_of_shares: int = Field(
+        ...,
+        title="Number of shares",
+        description="The total number of the shares in the share series class",
+        example=100,
+    )
+    share_value: float = Field(
+        ...,
+        title="Share value",
+        description="The total nominal value of the issued shares of the company",
+        example=10,
+    )
+    share_value_currency: Optional[ISO_4217_CurrencyCode] = Field(
+        None,
+        title="Share value currency",
+        description="The currency used for paying the settlement deposit in ISO 4217 format",
```

Comment:
@hsuomi: This description looks like a copy&paste mistake (it's as in the spreadsheet though)

Reply:
True. Updated in the spreadsheet. Should be "The currency used for the share value in ISO 4217 format"